src/Mod/Measure/Gui/TaskMeasure.cpp

```diff
@@ -100,17 +112,19 @@ TaskMeasure::TaskMeasure()
 
     formLayout->addRow(tr("Mode:"), modeSwitch);
     formLayout->addRow(showDeltaLabel, showDelta);
+    formLayout->addRow(autoSaveLabel, autoSaveCheckBox);
     formLayout->addRow(tr("Result:"), valueResult);
     layout->addLayout(formLayout);
+    auto* endLayout = new QHBoxLayout();
+    endLayout->addWidget(
+        new QLabel(tr("Use CTRL modifier to add a selection to the current measurement.")));
```

Comment:
IMO having to write this is a sign that it might not be ideal UX. I'd prefer an option to switch between greedy/normal selection and add that comment as a tooltip for that option.